client/device/asa/update.go

```diff
@@ -86,7 +86,17 @@ func Update(ctx context.Context, client http.Client, updateInp UpdateInput) (*Up
 				return nil, err
 			}
 
-			if err := retry.Do(ctx, asaconfig.UntilStateDone(ctx, client, asaReadSpecOutp.SpecificUid), retry.DefaultOpts); err != nil {
+			if err := retry.Do(
+				ctx,
+				asaconfig.UntilStateDone(ctx, client, asaReadSpecOutp.SpecificUid),
+				retry.NewOptionsBuilder().
+					Message("waiting for update ASA credentials to be done").
```

Comment:
Waiting for ASA credentials to be updated on CDO...

Reply:
done 👍